instance/testing.py

```diff
@@ -76,9 +76,9 @@
 SMS_DLT_ENTITY_ID = environ.get('SMS_DLT_ENTITY_ID')
 
 #: Twilio support for non-indian numbers
-SMS_TWILIO_SID = environ.get('SMS_TWILIO_SID')
-SMS_TWILIO_TOKEN = environ.get('SMS_TWILIO_TOKEN')
-SMS_TWILIO_FROM = environ.get('SMS_TWILIO_FROM')
+# SMS_TWILIO_SID = environ.get('SMS_TWILIO_SID')
+# SMS_TWILIO_TOKEN = environ.get('SMS_TWILIO_TOKEN')
+# SMS_TWILIO_FROM = environ.get('SMS_TWILIO_FROM')
```

Comment:
All `environ.get` lines can be deleted, as that happens automatically if the env var is named `FLASK_*`